TrafficCapture/coreUtilities/src/main/java/org/opensearch/migrations/tracing/BaseSpanContext.java

```diff
@@ -14,14 +15,17 @@ public abstract class BaseSpanContext<S extends IInstrumentConstructor>
     @Getter
     protected final S rootInstrumentationScope;
     @Getter
-    final long startNanoTime;
+    final long startTimeNano;
+    @Getter
+    final Instant startTimeInstant;
     @Getter
     Throwable observedExceptionToIncludeInMetrics;
     @Getter
     private Span currentSpan;
 
     protected BaseSpanContext(S rootScope) {
-        this.startNanoTime = System.nanoTime();
+        this.startTimeNano = System.nanoTime();
+        this.startTimeInstant = Instant.now();
```

Comment:
Pretty much.  startTime is for logs correlation.  I've added these comments to the base interface to explain...
```
public interface IWithStartTimeAndAttributes extends IInstrumentationAttributes {
    /**
     * This is used to calculate the precise duration of the span.  System.nanoTime() is guaranteed to be monotonic
     * and not susceptible to clock fluctuations due to system time resets
     */
    long getStartTimeNano();
    /**
     * This is by some ContextTrackers to log the recorded wall-time so that it can be easier to find the event
     * within logs. Notice that if the system clock is reset (which should be rare), there could be duplicated
     * values at different points in time.
     */
    Instant getStartTimeInstant();
```